frontend/src/components/SettingStringsEditor.tsx

```diff
@@ -12,6 +12,13 @@ export default function SettingStringsEditor() {
 
     async function handleSaveSettings() {
         try {
+            let existingNetwork = getExistingSettings().network;
+            if (existingNetwork !== nodeManagerSettings.network) {
+                if (window.confirm("Changing networks will delete your node's state. This can't be undone!")) {
+                    localStorage.clear();
+                    window.location.reload();
```

Comment:
Do we need to write to storage the network on confirm?

Reply:
Yeah I think after localStorage.clear() it needs to write network.

Reply:
yeah this happens in the `setup` function